converter.py

```diff
@@ -95,13 +103,13 @@ def get_dump_name(filename, extension):
             data = processor(data)
         logger.info('Converting {} ({} entries)'.format(
             fname,
-            len(data) if isinstance(data, list) else 1,
+            len(data) if isinstance(data, list) else int(data is not None),
         ))
 
         dump_name = get_dump_name(fname, new_ext)
-        jsonizer.dump(data, get_dump_name(fname, new_ext))
-        if dump_name != fname:
-            os.remove(fname)
-
-        n += 1
+        if data is not None:
```

Comment:
add an error message or something that it failed to load any model from said file... even though there might come from the serializer it would be nice to have it here too.